src/test/CMakeLists.txt

```diff
@@ -1776,8 +1778,7 @@ if(${WITH_CEPHFS})
   )
   target_link_libraries(test_c_headers
     librados
-    rados
-    libcephfs
+    cephfs
     ${EXTRALIBS}
```

Comment:
Oops, missed this because my test stuff was failing earlier because I had rbd disabled (and some src/test stuff doesn't build with rbd disabled currently, unreleated)